packages/monaco-editor-wrapper/index.html

```diff
@@ -33,7 +33,7 @@
 }`;
         let toggleDiff = true;
 
-        function startEditor() {
+        async function startEditor() {
```

Comment:
Is async here necessary? It doesn't `await` anything.

Reply:
No, you are right. It can be removed from `startEditor` and `swapEditor`.